PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,12 +6,12 @@ Reviewing pull requests takes a lot of time and we're all volunteers. Please mak
 
 ### Checklist
 
-- [ ] I have rebased properly. Please see [our tutorial on rebasing](http://coala.readthedocs.io/en/latest/Developers/Git_Basics.html#rebasing).
-- [ ] I have gone through the [commit guidelines](http://coala.readthedocs.io/en/latest/Developers/Writing_Good_Commits.html) and I've followed them.
+- [ ] I have rebased properly. Please see [our tutorial on rebasing](http://docs.coala.io/en/latest/Developers/Git_Basics.html#rebasing).
+- [ ] I have gone through the [commit guidelines](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html) and I've followed them.
```

Comment:
Please update the url to http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html